Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -685,7 +685,7 @@ public static int CalculateAttackDamage(DaggerfallEntity attacker, DaggerfallEnt
                 // Handle poisoned weapons
                 if (damage > 0 && weapon.poisonType != Poisons.None)
                 {
-                    InflictPoison(target, weapon.poisonType, false);
+                    InflictPoison(attacker, target, weapon.poisonType, false);
```

Comment:
Why was this changed to attacker? Poison weapons assign their payload to the target hit by the weapon, not to the wielder.

Reply:
Yeah, I've only added extra info for mods. I thought adding the parameter there would be consistent with the other formulas. No change in behavior for DFU itself

Reply:
All good, sorry about that. Still working through, but looks good so far. :)